index.js

```diff
@@ -97,6 +97,23 @@ class ScmBase {
         return Promise.reject(new Error('Not implemented'));
     }
 
+    /**
+     * Get the branch list which related the repository
```

Comment:
Get the branch list ~which~ related `to` the repository

Reply:
Thank you! I will fix.